### ISSUE TITLE
discouraged-ns does not use the level correctly

### ISSUE THREAD

**Issue by AndreaCrotti** [clj-kondo/clj-kondo#2357]
With this sort of configuration
```{:linters
 {:discouraged-namespace {clj-time.core {:message "Use java.time instead"
                                         :level :error}}}}
```
the level is still warning, see this repository to reproduce it.
